dashboard/api/pkg/server/client.go

```diff
@@ -112,6 +116,10 @@ func runStableDiffusion(prompt string, testing bool) (string, error) {
 	} else {
 		s = realSpec
 	}
-	s.Docker.Entrypoint = append(s.Docker.Entrypoint, prompt)
+	// TODO this is really gross, we can probably do better.
+	if entryPoint, ok := s.EngineSpec.Spec[model.DockerEngineEntrypointKey].([]string); ok {
+		entryPoint = append(entryPoint, prompt)
+		s.EngineSpec.Spec[model.DockerEngineEntrypointKey] = append(s.EngineSpec.Spec[model.DockerEngineEntrypointKey].([]string), entryPoint...)
+	}
```

Comment:
The alternative is to decode the `EngineSpec.Spec` map into its corresponding structure for field inspection. It's less efficient (hardly), but possibly more readable. thoughts?